gatsby-config.js

```diff
@@ -7,6 +7,7 @@ const contentfulConfig = {
   accessToken:
     process.env.CONTENTFUL_ACCESS_TOKEN ||
     process.env.CONTENTFUL_DELIVERY_TOKEN,
+    environment: process.env.CONTENTFUL_ENVIRONMENT || 'master'
```

Comment:
@axe312ger idea. What if we enabled this in the source plugin instead of at the starter level?